### PR TITLE
Ipad turns fix in Navigation

### LINKED ISSUE #4890: Next turn direction icon is not drawn on iPad
iPhone - Working | iPad - Nonworking
---|---
image|image

### PR DESCRIPTION
**Pull request by biodranik**
Fixes #4890

### CODE REVIEW

File: iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
-
-CGFloat const kSearchButtonsViewHeightPortrait = 200;
-CGFloat const kSearchButtonsViewWidthPortrait = 200;
-CGFloat const kSearchButtonsViewHeightLandscape = 56;
-CGFloat const kSearchButtonsViewWidthLandscape = 286;
-CGFloat const kSearchButtonsSideSize = 44;
-CGFloat const kBaseTurnsTopOffset = 28;
-CGFloat const kShiftedTurnsTopOffset = 8;
+constexpr CGFloat kTurnsiPhoneWidth = 96;
+constexpr CGFloat kTurnsiPadWidth = 140;
```
Comment (vng) [inline iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm:19]: nit: If you have changed here, I'd put { from the new line and put constexpr on the right as below.

**Review by vng — APPROVED** [review]

File: iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm [hunk 2]
```diff
@@ -15,16 +15,16 @@
 #include "geometry/angles.hpp"
 
 namespace {
-CGFloat const kTurnsiPhoneWidth = 96;
```
Comment (rtsisyk) [inline iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm:18]: Sorry, how is this related to the declared goal of PR?

**Review by rtsisyk — APPROVED** [review]
Screenshots look good to me

File: iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm (hunk 2 above)
Comment (biodranik) [reply iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm:18]: It doesn't harm, and it shows to others how the new code should be written.

File: iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm (hunk 1 above)
Comment (biodranik) [reply iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm:19]: ObjC braces start on the same line, but ok )

**Pull request merged**